reference-implementation/web-platform-tests/readable-streams/bad-strategies.js

```diff
@@ -19,9 +19,10 @@ test(() => {
 
 }, 'Readable stream: throwing strategy.size getter');
 
-test(() => {
+promise_test(() => {
 
-  const theError = new Error('a unique string');
+  const controllerError = new Error('a unique string');
```

Comment:
Can you use `{ name: 'error 1' }` and `{ name: 'error 2' }` instead, which would allow us to use `assert_throws` and `promise_rejects`?